plugins/kapt3/kapt3-compiler/src/org/jetbrains/kotlin/kapt3/stubs/ClassFileToSourceStubConverter.kt

```diff
@@ -751,10 +751,24 @@ class ClassFileToSourceStubConverter(val kaptContext: KaptContextForStubGenerati
         val value = field.value
 
         val origin = kaptContext.origins[field]
+        // Static fields in interfaces needs an initializer, even when just processing it using APT
```

Comment:
nit: Add "... as static initializes are not allowed in interfaces"

Reply:
Added